src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.ios.js

```diff
@@ -20,8 +21,10 @@ function CreatePollContainer({
   selectedtime = {day: 1, hour: 0, minute: 0},
   ontimechanged = () => {},
   polls,
-  expiredobject = {day: 7, hour: 24}
+  expiredobject = {day: 7, hour: 24},
+  isAnonimity = false
 }) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonimity` prop has been added to the component, and it's being used to determine the dynamic color. However, there is no default value provided for this prop. This could potentially lead to issues if the prop is not passed in when the component is used. To avoid potential issues, consider providing a default value.
```diff
-  isAnonimity = false
+  isAnonimity = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->